app/url/provider.go

```diff
@@ -246,6 +246,9 @@ func (p *provider) RegistryURL(_ context.Context, params ...string) string {
 	}
 
 	segments := []string{u.Path}
+	if len(params) > 0 {
```

Comment:
looks good.
params[0] shouldn't contain any space, but to make it robust, can consider:
```
if len(params) > 0  {
    params[0] = strings.ToLower(strings.TrimSpace(params[0]))
}
```

Reply:
thanks. spaces will not be there in the account/space in any flow, it's safe.